.github/workflows/README.md

```diff
@@ -0,0 +1,38 @@
+# Decidim GitHub Actions workflows
+
+We use GitHub Actions as CI.
+
+- `lint_code.yml`: runs the linters for Ruby, JS and ERB files.
+- `ci_main.yml`: runs the tests for the main folder
+- `ci_core.yml`: Runs the tests for eh `decidim-core` module. This workflow serves as a template for the rest of modules, since setup is mostly the same. In order to make changes, `cd` into the workflows folder and run `ruby generate_workflows.rb`.
```

Comment:
Maybe I'm not doing this right, but I don't see the `generate_workflows.rb` anywhere: 

```bash
$ cd .github/workflows/
$ ruby generate_workflows.rb
Traceback (most recent call last):
ruby: No such file or directory -- generate_workflows.rb (LoadError)
```

Reply:
Oh yes, I removed it because it was only useful to generate some workflows. After that, customizing them got too tricky with the Ruby script, so I resorted to find & replace directly.

I'll update the docs.

Reply:
Done!

Reply:
Perfect!